.github/workflows/pr-actions.yml

```diff
@@ -58,8 +58,8 @@ jobs:
     name: "Test: End to End"
     runs-on: ubuntu-latest
     needs: test-unit
-    # stop the job if it runs over 20 minutes to prevent a hanging process from eating minutes
-    timeout-minutes: 20
+    # stop the job if it runs over 60 minutes to prevent a hanging process from eating minutes
+    timeout-minutes: 60
```

Comment:
We updated this in main as well in a separate PR, so just merge main into your branch and take the 'main' version of this file instead

Reply:
Looks like there's a bit of discrepancy in there now - https://github.com/cuttle-cards/cuttle/blob/b0f096887de357b0f29ffc179eb49c7595f113f3/.github/workflows/pr-actions.yml#L61-L62

I can make any changes in this pr if needed.